index.html

```diff
@@ -5,17 +5,19 @@
     <meta charset="UTF-8">
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- Add a `License` to your `README` file.